Submitting mailroom.py for Session 05

UncleanlyCleric opened the pull request.

mdgregor commented:
Can you pull upstream so these conflicts can be resolved.

mdgregor commented on students/jesse_miller/session05/mailroom.py:
```diff
@@ -1,184 +1,204 @@
+    print(f"\n{'-'*80}\n{{:17}} | {{:<19}} | {{:<15}} | {{:<19}}\n{'-'*80}"\
+    .format(headers[0], headers[1], headers[2], headers[3]))
+
+    for k_ey, v_alue in donors.items():
```
what is with the random `_`'s here?

UncleanlyCleric commented:
Oh, right.  I was trying to find something in the linter so I changed all the silly stuff it complained about, but forgot to change it back.  Fixed now.

> On Feb 16, 2019, at 10:04 AM, Michael Gregor <notifications@github.com> wrote:
> 
> @mdgregor approved this pull request.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/UWPCE-PythonCert-ClassRepos/GP_Python210B_Winter_2019/pull/105#pullrequestreview-204528625>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AX5ta9fZgEoRjtyy8VzlIE7FEQqu_1jXks5vOEg6gaJpZM4a-hu6>.
>
